docs/security/access-control.md

```diff
@@ -0,0 +1,160 @@
+
+# Azure Resources Access Control
```

Comment:
Will be great to provide a short overview or description of scenarios this doc is meant to address.

Reply:
Thanks, I added a short overview.